.github/workflows/pull-request.yml

```
@@ -257,6 +257,120 @@
+        run: |
+          echo "=== Seeding Sample Data ==="
+          if docker exec talawa-api-1 /bin/bash -c 'set -a; source ./.env; set +a; pnpm run add:sample_data'; then
+            echo "✅ Seeding completed successfully"
+          else
+            echo "❌ Seeding failed"
+            exit 1
+          fi
+
+      - name: Checkout Flutter App Repository
+        uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)